schema/ont_papers.ttl

```diff
@@ -0,0 +1,114 @@
+@prefix : <https://w3id.org/ocs/ont/papers#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix fabio: <http://purl.org/spar/fabio/> .
+@prefix ocs_papers: <https://w3id.org/ocs/ont/papers/> .
+@base <https://w3id.org/ocs/ont/papers> .
+
+<https://w3id.org/ocs/ont/papers> rdf:type owl:Ontology ;
+                                   rdfs:comment "This ontology describes properties used in OpenCS knowledge graph about research articles from the domain of Computer Science."@en ;
+                                   rdfs:label "OpenCS Knowledge Graph ontology"@en .
+
+#################################################################
+#    Object Properties
+#################################################################
+
+###  https://w3id.org/ocs/ont/papers/hasRelated
+ocs_papers:hasRelated rdf:type owl:ObjectProperty ,
+                               owl:InverseFunctionalProperty ,
+                               owl:IrreflexiveProperty ;
+                      rdfs:domain fabio:ResearchPaper ;
+                      rdfs:range ocs_papers:RelatedObjectRelation ;
+                      rdfs:comment "Represents a relation between research paper and other object. Points to RelatedObjectRelation class. Other properties are then used to further describe the relation."@en .
```

Comment:
Please add `rdfs:label` for each class and property (for example: "Has related" for this prop). Please remember to add the labels with the "en" language tag – you can specify it in the bottom-right of the popup window in Protege when editing the value of some property.